.github/workflows/check.yml

```
@@ -0,0 +1,15 @@
+on: [push, pull_request]
+name: CI Checks
+jobs:
+  check:
+    name: Check and Lint
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@master
```

Using a fixed version to ensure nothing breaks in the future.

```suggestion
      - uses: actions/checkout@v2
```